src/functionalTest/java/uk/gov/hmcts/reform/judicialapi/client/S2sClient.java

```diff
@@ -52,7 +52,7 @@ public String signIntoS2S() {
 
         assertThat(response.getStatusCode()).isEqualTo(200);
 
-        String jwtToken = response.getBody().asString();
-        return jwtToken;
+        return response.getBody().asString();
```

Comment:
I think this was useful for debugging purposes - but I suppose it can be changed when developer is debugging.